### DISCUSSION TITLE
I just used discord after 1 year

### DISCUSSION THREAD

**Discussion by amir-khoshbakht** [discordjs/discord.js#8827] [General]
I have been familiar with discord software for years. I used it again today after a period of 1 year. I can't believe what happened to it. Every time you open a section, a damn message appears and asks, do you want to try this? no I dont want.
Is there a setting to never ask me this stupid question again? Do you want to try this? I don't want to try anything new.

**Comment by almostSouji** [comment]
Discord.js is a Discord bot API wrapper, we are not Discord and have no control over any of this, nor can the library help you with user accounts or client modifications.
